common/tzif.ksy

```diff
@@ -0,0 +1,256 @@
+meta:
+  id: tzif
+  title: The Time Zone Information Format (TZif)
+  xref:
+    mime: application/tzif
+    # mime: "application/tzif-leap" if leap-second records are included
+    rfc: 8536
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: be
+doc: |
+  TZif is the binary file format used to represent data for a single time zone
+  compiled by the _zic_ compiler from information in the IANA time zone
+  database. It is used by most UNIX systems to calculate local time for a time
+  zone, usually identified by a time zone identifier string referring to a
+  relatively populous location within the geographic region covered by the time
+  zone, for example: "America/New_York" or "Europe/Paris". This identifier is
+  external to the TZif file itself.
```

Comment:
It might be good to include a link to the specifications. The file format is described in section 5 of the manpage of tzfile:

```
man 5 tzfile
```

with an online reference here:

http://man7.org/linux/man-pages/man5/tzfile.5.html

Reply:
The ultimate specification is RFC 8536. I tagged that in meta/xref/rfc. I could add a doc-ref linking to the html version of the RFC. I'm reluctant to link to the Linux manpage because it is not definitive. The source for the definitive man page is in the tzcode or tzdb package, but I don't know of a nice, formatted version online.

Reply:
I've added a doc-ref to the top level pointing to the html version of the RFC, and added some other doc-ref lines pointing to sections of the RFC where appropriate.

Reply:
I guess, if it is in xrefs, it shouldn't be in doc-refs. @generalmimon, what do you think?

Reply:
@KOLANICH:

> I guess, if it is in xrefs, it shouldn't be in doc-refs. @generalmimon, what do you think?

I agree with you, there is no need for that. At least the top-level `/doc-ref` is useless given that it's already in `/meta/xref`. `doc-ref`s in subtypes pointing to a specific section with a hash in the URL (like `https://tools.ietf.org/html/rfc8536#section-3.1`) kind of make sense because they provide additional information, and you don't have to find the relevant section in the entire RFC spec.

Reply:
Removed the top-level `/doc-ref`.